packages/salesforcedx-vscode-apex/src/commands/forceApexLogGet.ts

```diff
@@ -129,7 +129,7 @@ export class ApexLibraryGetLogsExecutor extends LibraryCommandletExecutor<{
     const logPath = logResults[0].logPath;
     if (logPath) {
       const document = await vscode.workspace.openTextDocument(logPath);
-      vscode.window.showTextDocument(document);
+      void vscode.window.showTextDocument(document);
```

Comment:
What's the purpose of this `void` declaration?

Reply:
The call to `vscode.window.showTextDocument(document);` returns a promise and the run function is async. We have an eslint rules that currently warns of a situation where the code is not properly handling the called functions promise/then-able return. Since this is opening a text document and the previous code was not capturing the returned promise I assume here that the intent was to open the text document w/o concern of the result.

To resolve the eslint and more importantly acknowledge in the code to match the intent, one can use void to say I am not concerned about the result, so call the function and ignore the presence of the returned promise.

The called function will run and resolve on its own and the calling function will continue without having to handle the promise. General term would be "fire the event and forget the result."

Reply:
Got it!  Thank you for the explanation - I'm clear now 👍